orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/metrics/ZombieExecutionCheckingAgent.kt

```diff
@@ -40,7 +40,7 @@ import org.springframework.stereotype.Component
 @ConditionalOnExpression(
   "\${queue.zombie-check.enabled:false}"
 )
-@ConditionalOnBean(MonitorableQueue::class)
+@ConditionalOnBean(ZombieExecutionService::class)
```

Comment:
i think you still need the conditional on queue since it's used by this class?

Reply:
Is it?